src/DSP/Filter.cpp

```diff
@@ -59,8 +59,10 @@ Filter *Filter::generate(Allocator &memory, const FilterParams *pars,
             filter->setgain(pars->getgain());
             break;
         case 4:
-            filter = memory.alloc<CombFilter>(&memory, Ftype, 1000.0f, pars->getq(), srate, bufsize);
+            {
+            filter = memory.alloc<CombFilter>(&memory, Ftype, 1000.0f, pars->getq(), srate, bufsize, pars->Plpf, pars->Phpf);
             filter->setgain(pars->getgain());
+            }
```

Comment:
Thanks to your successful refactor we should be able to remove the `{` on line 62 and the `}` on 65 :)